samples/bluetooth/hci_spi/boards/nrf52dk_nrf52832.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
Is this correct? Is the file copied from somewhere?

Reply:
yes the reference file is nrf51dk_nrf51422 overlay, I just modified the pins for nrf52dk_nrf52832